pkg/kubernetes/scripts/virtual-machines/components/VmMetricsTab.jsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// @flow
+import React, {PropTypes} from 'react';
+import { DonutChart } from 'patternfly-react'
+import { gettext as _ } from 'cockpit';
+import cockpit from 'cockpit';
+
+import './VmMetricsTab.less';
+
+React;
```

Comment:
Not needed anymore since we switched to ESLint.
But feel free to keep it if no more force-push is planed.